packages/graphics/src/const.ts

```diff
@@ -13,12 +21,13 @@
  * @property {number} minSegments=8 - minimal number of segments in the curve (if adaptive = false, ignored)
  * @property {number} maxSegments=2048 - maximal number of segments in the curve (if adaptive = false, ignored)
  */
-export const GRAPHICS_CURVES = {
+export const GRAPHICS_CURVES: IGraphicsCurvesSettings = {
     adaptive: true,
     maxLength: 10,
     minSegments: 8,
-    maxSegments: 2048,
-    _segmentsCount(length, defaultSegments = 20)
+    maxSegments:  2048,
+
+    _segmentsCount(length: number, defaultSegments = 20)
```

Comment:
`private _segmentsCount(length: number, defaultSegments = 20): number`

Reply:
We can't do it.
It used on ArcUtils.